mne/io/pick.py

```diff
@@ -846,11 +846,12 @@ def _check_excludes_includes(chs, info=None, allow_bads=False):
     misc=False, resp=False, chpi=False, exci=False, ias=False, syst=False,
     seeg=True, dipole=False, gof=False, bio=False, ecog=True, fnirs=True)
 _PICK_TYPES_KEYS = tuple(list(_PICK_TYPES_DATA_DICT.keys()) + ['ref_meg'])
-_DATA_CH_TYPES_SPLIT = ('mag', 'grad', 'eeg', 'seeg', 'ecog', 'hbo', 'hbr')
+_DATA_CH_TYPES_SPLIT = ('mag', 'grad', 'eeg', 'seeg', 'ecog', 'hbo', 'hbr',
+                        'ref_meg')
 _DATA_CH_TYPES_ORDER_DEFAULT = ('mag', 'grad', 'eeg', 'eog', 'ecg', 'emg',
-                                'reg_mag', 'misc', 'stim', 'resp', 'chpi',
-                                'exci', 'ias', 'syst', 'seeg', 'bio', 'ecog',
-                                'hbo', 'hbr', 'whitened')
+                                'reg_mag', 'ref_meg', 'misc', 'stim', 'resp',
```

Comment:
I was unsure if 'reg_mag' was a typo so I left it.

Reply:
It's a typo